File: Leetcode75/Day 2 String/Q392_Is_Subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if len(s)==0:
            return True
        for i in t:
            if s[0]==t[0]:
                s=s[1:]
            t=t[1:]
            if len(s)==0:
                return True
        return False

File: Leetcode75/Day 2 String/Q392_is_Subsequence.java
public class Q392_is_Subsequence{
    public boolean isSubsequence(String s, String t){
        if (s.length()==0){
            return true;
        }
        int len = t.length();
        for (int i=0;i<len;i++){
            if(s.charAt(0)==t.charAt(0)){
                s=s.substring(1);
            }
            t=t.substring(1);
            if (s.length()==0){
                return true;
            }
        }
        return false;
    }
}